Services/Repository/Favourites/classes/class.ilFavouritesListGUI.php

```diff
@@ -63,20 +63,34 @@ public function render(): string
             }
         }
         if (count($item_groups) > 0) {
+            $tpl = new ilTemplate("tpl.favourites.html", true, true, "Services/Repository/Favourites");
             $ctrl->setParameterByClass(ilSelectedItemsBlockGUI::class, "view", "0");
             $ctrl->setParameterByClass(ilSelectedItemsBlockGUI::class, "col_side", "center");
             $ctrl->setParameterByClass(ilSelectedItemsBlockGUI::class, "block_type", "pditems");
-
             // see PR discussion at https://github.com/ILIAS-eLearning/ILIAS/pull/5247/files
+            $roundtrip_modal = $this->ui->factory()->modal()->roundtrip(
+                $this->lng->txt('rep_configure'),
+                $this->ui->factory()->legacy('PH')
+            )->withAdditionalOnLoadCode(function ($id) {
+                return "document.body.appendChild(document.getElementById('$id'));";
+            });
+            $roundtrip_modal = $roundtrip_modal->withAsyncRenderUrl(
+                $this->ctrl->getLinkTargetByClass(
+                    [ilDashboardGUI::class, ilColumnGUI::class, ilSelectedItemsBlockGUI::class],
+                    'removeFromDeskRoundtrip'
+                ) . '&page=manage&replaceSignal=' . $roundtrip_modal->getReplaceSignal()->getId()
+            );
```

Comment:
The implementation inside the modal is part of the Dashboard service (ilSelectedItemsBlockGUI). Please move these lines to the Dashboard service, too.